Sources/SmileID/Classes/Helpers/LocalStorage.swift

```diff
@@ -20,12 +22,30 @@ public class LocalStorage {
         }
     }
 
+    static var pendingDirectory: URL {
+        get throws {
+            try defaultDirectory.appendingPathComponent(pendingFolderName)
+        }
+    }
+
+    static var completedDirectory: URL {
+        get throws {
+            try defaultDirectory.appendingPathComponent(completedFolderName)
+        }
+    }
+
+    enum SaveType {
+        case pending
+        case completed
+    }
+
+    /// todo - rework this logic
     static func saveImage(
         image: Data,
         to folder: String = "sid-\(UUID().uuidString)",
```

Comment:
Is it possible to remove this parameter right now since it has a default value? If not we can do in the later PRs as you indicated